EXT_BeamPhysics.md

```diff
@@ -0,0 +1,269 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================
+
+Version 1.0.0 (March 1, 2018)
```

Comment:
Agreed

Reply:
@RemiLehe On second thought I believe we do need the version number since how else do we know what version of the Standard a given version of a file is associated with? Of course the version number here should be 2.0.0. @ax3l: Do you agree?

Reply:
Just updated in another thread we discussed: we keep it in the main document for now.

Reply:
Axel asked for removal so the version has been removed.